src/components/calcite-button/readme.md

```diff
@@ -4,6 +4,11 @@ You can programmatically focus a `calcite-button` with the `setFocus()` method:
 
 `<div onclick=document.querySelector('#my-button').setFocus()>Focus!</div>`
 
+Any additional attributes set on `<calcite-button>` are passed to the internal `<a>` or `<button>` tag. For example:
+
+- `<calcite-button href="https://github.com/esri/calcite-components target="_blank">Calcite Components on GitHub</calcite-button>` would set `target="_blank` On the internal `<a>`.
```

Comment:
This would be good as a file in `/calcite-button/usage/internal.md` instead of directly in the readme.

Reply:
@driskull @macandcheese I feel like we should document all the functionality in the readme. I probably wouldn't have thought to dig into a `usage/internal.md` file looking for details on how component internals work for basic usage.

Reply:
@patrickarlt any markdown files in the `/usage/` folder get compiled into the readme under the `# Usage` section with a subsection matching the file name. This encourages consistency for usage throughout component doc